apps/site/src/views/Home/sections/Intro/Intro.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "bootstrap-utils" as bootstrap;
+
+.intro {
+    background-image: url("~src/assets/images/index-card.png");
+    background-size: 100% 100%;
```

Comment:
Note: the index card line spacing looks comically large on narrower views. We may need to consult with graphics.

Reply:
For now, we should close this issue and open a new one once graphics is done with the index card fixes.